src/deployment/registration.py

```diff
@@ -211,6 +211,28 @@ def create_application_registration(
 
 
 def add_application_password(app_object_id: UUID) -> Tuple[str, str]:
+    # Work-around the race condition where the app is created but passwords cannot
```

Comment:
Do we have any generic exponential backoff code we could use in this function?

Reply:
We don't at this time.  We could use something like that. I'll work on that in the future.